documentation/asciidoc/accessories/camera/timelapse.adoc

```diff
@@ -69,7 +69,7 @@ On a Raspberry Pi 3, this can encode a little more than two frames per second. T
 
 * `-r 10` Set frame rate (Hz value) to ten frames per second in the output video.
 * `-f image2` Set ffmpeg to read from a list of image files specified by a pattern.
-* `-pattern_type glob` When importing the image files, use wildcard patterns (globbing) to interpret the filename input by `-i`, in this case "image__.jpg", where "__" would be the image number.
+* `-pattern_type glob` When importing the image files, use wildcard patterns (globbing) to interpret the filename input by `-i`, in this case `image*.jpg`, where `*` would be the image number.
```

Comment:
Ping @davidplowman. Could you take a look at this?

Reply:
I'm not sure this bit needs checking? The example command-line a few lines above says:
```
ffmpeg -r 10 -f image2 -pattern_type glob -i 'image*.jpg' -s 1280x720 -vcodec libx264 timelapse.mp4
```

Reply:
Ah! Gotcha.